.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -11,15 +11,16 @@ body:
       1. Please speak English, this is the language all maintainers can speak and write.
       2. Please ask questions or configuration/deploy problems on our Discord
          server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
-      3. Please take a moment to check that your issue doesn't already exist.
+      3. Make sure you are using the latest release and
+         take a moment to check that your issue hasn't been reported before.
```

I'd say these two can be separate points:
```suggestion
      3. Please make sure you are using the latest release
      4. Please take a moment to check that your issue hasn't been reported before.
```

---

There are already many `please`s above and below. I don't like to put too many pleases. `Make sure` is clear enough